### DISCUSSION TITLE
How to optimize index on replication tables (merge publish)

### DISCUSSION THREAD

**Discussion by Stocki916** [olahallengren/sql-server-maintenance-solution#451] [Questions & Answers]
Hi,

we are running a merge publish replication in sql server and the replication uses some system tables (with prefix "MSMerge_") which are created inside the replicated user database as system tables.
So if I execute the index rebuild script with parameter "@Databases=USER_DATABASES" only the user created tables in each database are used to optimize its indizes. If I use "@Databases=SYSTEM_DATABASES" only the master, model etc databases are used to optimize indizes. 
Is there any chance to get the replication indizes in user database?

best regards,
sebastian

**Comment by olahallengren** [comment]
Could you try with the parameter ```@MSShippedObjects = 'Y'```?
https://ola.hallengren.com/sql-server-index-and-statistics-maintenance.html#MSShippedObjects
